src/KingSystem/Framework/GameConfig.h

```diff
@@ -1,9 +1,15 @@
+#pragma once
+#include <prim/seadSafeString.h>
+
 class GameConfig {
 public:
     static GameConfig* getInstance() { return sInstancePtr; }
 
-    char mPadding[0x419];
+    char mPad0[0x3dd];
```

Comment:
We typically use names like `_0` (underscore + offset in lowercase hex) to make it clear what the offset is and for less typing.